test/Frontend/immediate-mode-no-output.swift

```diff
@@ -0,0 +1 @@
+// RUN: rm -rf %t &&  mkdir -p %t && cd %t && echo >test.swift && %swift  -interpret test.swift &&  rm test.swift && test `ls | wc -w` -eq 0
```

Comment:
Here's my cleanup of this test, but the actual substance looks fine!

```
// RUN: %empty-directory(%t)
// RUN: cd %t && %target-swift-frontend -interpret %S/../Inputs/empty.swift
// RUN: not ls %t/*
```

- `%empty-directory` is shorthand for the `rm -rf`+`mkdir` invocation (just so that everyone does it in a standard way).
- IIRC the `cd` still has to be on the same line because every line runs in its own subshell.
- `test/Inputs/empty.swift` is a nice designated "empty Swift file"; unfortunately you have to refer to it by relative path.
- Slightly simpler testing for an empty directory.

Reply:
Cool, thanks!

Reply:
Oops, forget that you need one more line, `// REQUIRES: swift_interpreter`, so that we don't try to run it on platforms that don't support the interpreter.

Reply:
Thanks. Done!